src/GraphQL/Federation/Types/LinkImportGraphType.cs

```diff
@@ -0,0 +1,15 @@
+namespace GraphQL.Federation.Types;
+
+/// <summary>
+/// Represents a scalar type for link imports in GraphQL Federation.
+/// Used to define imported directives.
+/// The name of this graph type is "link__Import".
+/// </summary>
+public class LinkImportGraphType : AnyScalarGraphType
```

Comment:
Why does it derive from Any and not String?

Reply:
https://specs.apollo.dev/link/v1.0/#Import

> [Import](https://specs.apollo.dev/link/v1.0/#Import) can take the form of a string ... or an object with name and (optionally as)

Reply:
Shouldn't it have a validation for these two forms only?